src/ngx_http_lua_common.h

```diff
@@ -538,6 +534,7 @@ typedef struct ngx_http_lua_ctx_s {
     unsigned         seen_body_data:1;
 
     ngx_uint_t       cur_rewrite_index;
+    ngx_uint_t       cur_access_index;
```

Comment:
I'm afraid it's wrong to add word-sized fields after bit-sized fields. It creates potential padding holes in the middle, which wastes some space.